### PR TITLE
Fix remaining segmentation fault that occurs with the patched QComboBox in PySide

### PR DESCRIPTION
**Pull request by astrofrog**
For now, I'm adding a regression test to make sure it fails on Travis/AppVeyor. This includes #37 and should be rebased once #37 is merged.

### CODE REVIEW

**Comment by astrofrog** [comment]
With this and #37, I'm able to get all the glue tests to pass!

**Comment by ccordoba12** [comment]
Nice!! So would it be ok to relase qtpy **1.1** after merging your three pending PRs? Or do you have something else to add?

**Comment by astrofrog** [comment]
@ccordoba12 - I think we should be good for a release once all 3 are merged - but let me confirm that once that's the case.

**Comment by astrofrog** [comment]
@ccordoba12 - would you be open to a PR for the README to mention that some of the code comes from qt-helpers from the glue project?

**Comment by ccordoba12** [comment]
> would you be open to a PR for the README to mention that some of the code comes from qt-helpers from the glue project?

Sure, no problem

> should I add my name to AUTHORS

Of course, please do it ;-)

**Comment by astrofrog** [comment]
@Nodd @ccordoba12 this is ready for a final review

**Comment by ccordoba12** [comment]
I don't have too much to say about this one. Given that tests pass, I'm +1 for it.

@Nodd, do you want to comment here?

**Comment by ccordoba12** [comment]
We don't officially support PySide anymore in Spyder, so that's why I'm neutral about this ;-)

File: tests/test_patch_qcombobox.py [hunk 1]
```diff
@@ -73,3 +74,27 @@ def __getitem__(self, item):
...
+    This is a regression test for an issue that caused the call to item(0)
+    below to trigger segmentation faults in PySide. The issue is
+    non-deterministic when running the call once, so we include a loop to make
+    sure that we trigger the fault.
+    """
+
+    app = get_qapp()
+
+    combo = QtWidgets.QComboBox()
+
```
Comment (Nodd) [inline tests/test_patch_qcombobox.py]: That's a lot of whitespace here, can you remove these blank lines ?

**Comment by Nodd** [comment]
Same as @ccordoba12, I think you're the only one using this feature right now so as long as it works for you, it should be ok !

**Comment by astrofrog** [comment]
@Nodd - done!

**Comment by Nodd** [comment]
Thank you, merging then !

**Pull request merged**